modules/plot/test/spec.coffee

```diff
@@ -1,8 +1,12 @@
+# Used to mimic an event call for a node
+fakeNodeEvent = (node, eventName) ->
+  if node?
+    (e) -> hx.select.getHexagonElementDataObject(node).eventEmitter?.emit((if eventName? and isNaN(eventName) then eventName else 'click'), e)
```

Comment:
That was my guess as well, that we could extract it to somewhere shared. I'm not sure how often it is already being used.
This is copied from the data-table spec (Charlie told me to do the same yesterday with the meter)